app/src/actions.js

```diff
@@ -32,6 +32,7 @@ export const UPDATE_FILTERS = 'UPDATE_FILTERS';
 
 // Appearance actions
 export const SET_VISIBLE_MENU = 'SET_VISIBLE_MENU';
+export const SUPPORT_MODAL_OPEN = 'SUPPORT_MODAL_OPEN';
```

Comment:
we probably don't need an action for this. This is view state, not application state. Nothing outside the map component even knows that there's a support modal window.